radp/digital_twin/mobility/ue_generation_handler.py

```diff
@@ -0,0 +1,257 @@
+        min_lon (float): Minimum longitude boundary.
+        max_lon (float): Maximum longitude boundary.
+        mobility_class_distribution (Dict[MobilityClass, float]): Distribution of mobility classes.
+        mobility_class_velocities (Dict[MobilityClass, float]): Average velocities for each mobility class.
+        mobility_class_velocity_variances (Dict[MobilityClass, float]): Variance of velocities for each mobility class.
+    """
+    
+    def __init__(
+        self,
+        rng_seed: int,
+        mobility_class_distribution: Dict[MobilityClass, float],
+        mobility_class_velocities: Dict[MobilityClass, float],
+        mobility_class_velocity_variances: Dict[MobilityClass, float],
+        num_batches : int = 1,
+        lon_x_dims: int = 100,
+        lon_y_dims: int = 100,
+        num_ticks: int = 2,
+        num_UEs: int = 2,
+        alpha: float = 0.5,
+        variance: float = 0.8,
+        min_lat: float = -90,
+        max_lat: float = 90,
+        min_lon: float = -180,
+        max_lon: float = 180,
+        ):
+
+        self.rng_seed = rng_seed
+        self.num_batches = num_batches
+        self.lon_x_dims = lon_x_dims
+        self.lon_y_dims = lon_y_dims
+        self.num_ticks = num_ticks
+        self.num_UEs = num_UEs
+        self.alpha = alpha
+        self.variance = variance
+        self.min_lat = min_lat
+        self.max_lat = max_lat
+        self.min_lon = min_lon
+        self.max_lon = max_lon
+        self.mobility_class_distribution = mobility_class_distribution
+        self.mobility_class_velocities = mobility_class_velocities
+        self.mobility_class_velocity_variances = mobility_class_velocity_variances
+
+
+
+    def generate_ue_data(self) -> List[Any]:
+
+        """
+        Generates the UE Tracks using the specified Gauss-Markov Mobility Parameters.
+        
+        return: A list containing the batches of all the UE Tracks data created.
+        """
+
+        ue_generator = UETracksGenerator(
+            rng=np.random.default_rng(self.rng_seed),
+            mobility_class_distribution=self.mobility_class_distribution,
+            mobility_class_velocities=self.mobility_class_velocities,
+            mobility_class_velocity_variances=self.mobility_class_velocity_variances,
+            lon_x_dims=self.lon_x_dims,
+            lon_y_dims=self.lon_y_dims,
+            num_ticks=self.num_ticks,
+            num_UEs=self.num_UEs,
+            alpha = self.alpha,
+            variance = self.variance,
+            min_lat = self.min_lat,
+            max_lat = self.max_lat,
+            min_lon = self.min_lon,
+            max_lon = self.max_lon
+            ) 
+
+        all_batches = []
+        for batch_no in range(self.num_batches):
+            batch = next(ue_generator.generate())
+            all_batches.append(batch)
+        print("Batches of Data: " , all_batches)
+        return all_batches
+
+
+
+    def mobility_data_generation(self) -> pd.DataFrame:
```

Comment:
this method and `generate_ue_data` can be replaced with the original `_mobility_data_generation`, which can actually be moved to this directory (as you are correctly doing in this PR), into `ue_tracks.py`, and we may rename it as `generate_as_lon_lat_points`, and then fix the call-site.

Reply:
Updated the ue_tracks by adding _mobility_data_generation from services/ue_tracks_generation/u_tracks_generation_driver.py file to ue_tracks.py. 
Renamed it to generate_as_lon_lat_points.

Will delete the reduntant ue_generation_handler.py after plot functions is fixed.